dev/sparktestsupport/modules.py

```diff
@@ -431,6 +431,7 @@ def __hash__(self):
         "pyspark.ml.linalg.__init__",
         "pyspark.ml.recommendation",
         "pyspark.ml.regression",
+        "pyspark.ml.stat",
```

Comment:
@holdenk  If we need to add pyspark.ml.stat to setup.py, then why are we not adding the other analogous modules: pyspark.ml.{classification, clustering, regression,...}?